.github/workflows/publish-draft-releases.yml

```diff
@@ -340,11 +340,403 @@ jobs:
       #   run: |
       #     jq --slurp '. | add' ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay-*.json > ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay.json
 
+  build-node-base:
+    needs:
+      - start-node-builder-base
+      - check-for-runtime-upgrade
+    runs-on: ${{ needs.start-node-builder-base.outputs.runner-label }}
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      -
+        if: ${{ needs.check-for-runtime-upgrade.outputs.runtime-version-current != needs.check-for-runtime-upgrade.outputs.runtime-version-base }}
+        uses: actions/checkout@v2
+        with:
+          ref: 'manta-pc'
+          # ref: 'f08ce6840a0ccfdeb6a34a70a28235a948aad85d'
```

Comment:
Not sure if we should leave this as `manta-pc` or let it be an input for the developer.

The commented hash is a commit which has the correct accounts for the chain spec, as ``manta-pc`` doesn't have them yet. Used it to trigger the last CI build.